### PR TITLE
General UX improvements WIP

### PR DESCRIPTION
**Pull request by yazadzhy**

### CODE REVIEW

File: frontend/src/ui/wallet/transfer/transfer-settings.js [hunk 1]
```diff
@@ -139,6 +139,7 @@ export default class TransferSettings {
             this.createTrustline = false
         }
         this.asset[1] = this.asset[0]
+        this.amount[1] = this.amount[0]
```
Comment (orbitlens) [inline frontend/src/ui/wallet/transfer/transfer-settings.js:142]: Could you please explain this?

File: frontend/src/ui/wallet/transfer/transfer-view.js [hunk 2]
```diff
@@ -52,28 +67,70 @@ function TransferView() {
...
+    function getAccountPredefinedDisplayName(address) {
+        if (!window.predefinedAccountDisplayNames) return undefined
+        return window.predefinedAccountDisplayNames[address]
+    }
+
+    async function saveNewAddress() {
+        const {address, ...predefinedAddress} = newAddress
+        predefinedAddress['network'] = network
+        predefinedAddress['federation_address'] = await getFederationAddress(transfer.destination)
+        predefinedAddress.memo.type = transfer.memo?.type || 'none'
```
Comment (orbitlens) [inline frontend/src/ui/wallet/transfer/transfer-view.js:79]: Do not create `predefinedAddress.memo` at all unless `transfer.memo.type` and `transfer.memo.value` set.

**Review by orbitlens — COMMENTED** [review]
Transfer-related comments

File: frontend/src/ui/wallet/transfer/transfer-settings.js (hunk 1 above)
Comment (yazadzhy) [reply frontend/src/ui/wallet/transfer/transfer-settings.js:142]: Amount need reset like asset, coz if form filled and just swaped tabs of transfer form (ex: convert -> claimable -> convert): selected assets is reseted, but amount - not.

1 XLM ~ 0.8 USDC and after swapping of tabs will be shown on form 1 XLM ~ 0.8 XML

**Pull request merged**